testdata/kata/catalogsource.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
- name:  qe-optional-operators
+ name:  redhat-operators
  namespace: openshift-marketplace
 spec:
  sourceType: grpc
- image: quay.io/openshift-qe-optional-operators/ocp4-index:latest
+ image: registry.redhat.io/redhat/redhat-operator-index:v4.9
```

Comment:
@valtair1891 this is hardcoded and won't scale for future releases.  Let's think this over a bit.

Reply:
Agreed, we can create a different catalog source file for each release, as an immediate solution, is it make sense?

Reply:
@valtair1891 @cpmeadors if the only difference is the `4.9` and the rest format stays the same, then you can make that portion as a variable that correspond to the OCP version.